pageObject/loginPage.js

```diff
@@ -13,10 +13,8 @@ class LoginPage extends Page {
         await this.loginBtn.click();
     }
 
-    async openSite() {
-        await browser.url(
-          "https://www.saucedemo.com/"
-        );
+    open() {
+        super.open(" ");
```

Comment:
Do we need space here? looks strange

Reply:
thanks! no omre space

open() {
    super.open("");
  }